src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php

```diff
@@ -124,8 +124,14 @@ public function send(string $body, array $headers, int $delay = 0): string
             $body,
             json_encode($headers),
             $this->configuration['queue_name'],
-            self::formatDateTime($now),
-            self::formatDateTime($availableAt),
+            $now,
+            $availableAt,
+        ], [
+            null,
+            null,
+            null,
+            Type::getType(Type::DATETIME),
```

Comment:
In https://github.com/symfony/symfony/pull/32456#discussion_r329623629 I asked to only pass the constant. No need to call `Type::getType` anywhere. Please it's very painful to review this PR over and over and the same problems keep being there.